ui/src/shared/components/SingleStat.tsx

```diff
@@ -121,6 +122,10 @@ class SingleStat extends PureComponent<Props> {
       cellType: lineGraph ? CellType.LinePlusSingleStat : CellType.SingleStat,
     })
 
+    if (onUpdateCellColors) {
```

Comment:
This guards against the situation in which a `RefreshingGraph` is not inside a `LayoutCell` and does not receive the function

Reply:
would it work to provide a default prop function that does nothing in that case?